packages/common/src/HardhatConfig.js

```diff
@@ -27,9 +27,9 @@ function getHardhatConfig(configOverrides, workingDir = "./") {
     },
     networks: {
       hardhat: {
-        gas: 11500000,
-        blockGasLimit: 11500000,
-        allowUnlimitedContractSize: false,
+        gas: 12000000,
+        blockGasLimit: 12000000,
+        allowUnlimitedContractSize: true,
```

Comment:
nit: why do we need this? If this is breaking, doesn't it signal that the contract is too big to deploy on mainnet (which we would want to show up in tests)?

Reply:
I actually think the main issue is that our gas limit is set too low and we should leave it undefined and let hardhat estimate it. The contract is well under the bytecode according to CalculateContractBytecode

Reply:
Ended up removing this as I realize that `PerpetualLib` was actually over bytecode limit, a problem that I fixed by removing require messages.

Reply:
Sounds good!